modules/nw-exposing-service.adoc

```diff
@@ -72,17 +72,47 @@ HTTP/1.1 200 OK
 
 endif::nodeport[]
 ifdef::nodeport[]
-. To expose a node port for the application, enter the following command. {product-title} automatically selects an available port in the `30000-32767` range.
+. To expose a node port application, modify the custom resource definition of a service by entering by the following command:
 +
 [source,terminal]
 ----
-$ oc expose service nodejs-ex  --type=NodePort --name=nodejs-ex-nodeport --generator="service/v2"
+$ oc edit svc <service_name>
 ----
 +
 .Example output
 [source,terminal]
 ----
-service/nodejs-ex-nodeport exposed
+spec:
+  ...
+  ports:
+  - name: 8080-tcp
+  nodePort: 30546
+  port: 8080
+  protocol: TCP
+  targetPort: 8080
+  - name: 8443-tcp
+  nodePort: 30327 <1>
```

Comment:
It seems the indentation is not correct for this yaml, please refer to the  bug.

Reply:
Updated the identation